csharp/src/Drivers/Apache/Spark/README.md

```diff
@@ -26,7 +26,7 @@ but can also be passed in the call to `AdbcDatabase.Connect`.
 
 | Property               | Description | Default |
 | :---                   | :---        | :---    |
-| `adbc.spark.type`      | (Required) Indicates the Spark server type. One of `databricks`, `http` (future: `standard`) | |
+| `adbc.spark.type`      | (Required) Indicates the Spark server type. Currently only `http` (future: `standard`) | |
```

Comment:
I don't think `standard` would apply to Databricks which is likely always/only `http`. (I seem to recall that `standard` is a TCP-based Thrift transport but might be misremembering.)

Reply:
Hmm. I read this as it only applies to the Spark library now (http, standard) and not for Databricks. Does it not read that way?

Reply:
Derp; for some reason I thought this was the Databricks-specific entry.